bitcoind-rpc-test/src/test/scala/org/bitcoins/rpc/v18/PsbtRpcTest.scala

```diff
@@ -130,10 +130,21 @@ class PsbtRpcTest extends BitcoindRpcTest {
     val nodeF = startF.flatMap(_ => client.getNodeAddresses())
 
     //Todo figure out how to check time and port
-    nodeF.map({ result =>
+    nodeF.map { result =>
       assert(result.head.address.isAbsolute)
       assert(result.head.services >= 0)
       assert(result.nonEmpty)
-    })
+    }
+  }
+
+  it should "return active rpc commands" in {

```

Comment:
This should probably be moved into `BitcoindV18RpcClientTest`, as it isn't related to PSBTs